apps/organisations/templates/a4_candy_organisations/communication_form_social_media.html

```diff
@@ -43,19 +64,16 @@ <h1>{% translate "Create Content" %}</h1>
                         </div>
                     {% endif %}
                 </div>
+                {% if image_preview %}
 
                 <div class="d-flex justify-content-end mb-5">
                     <button type="submit" class="btn btn--primary btn--icon">
-                        {% if image_preview %}
-                            <i class="fas fa-sync-alt"></i>{% translate 'Refresh' %}
-                        {% else %}
-                            {% translate 'Generate' %}
-                        {% endif %}
```

Comment:
Did you remove this on purpose @phillimorland ?

Reply:
@Rineee  yeah because otherwise there ware 2 generate buttons which didn't make sense right? unless I accidentally added another?!

Reply:
could that be what broke it? sorry when we talked about didn't mess with validation thought it was in the forms.py

Reply:
Noo, dont think thats not what broke it!
I meant to add a 'Generate' button when there is no preview yet and a 'Refresh' button, when there is an preview, think it also works on dev? https://aplus-dev.liqd.net/liqd-tests/dashboard/communication/content/create/multimodul-test/format/1/

Reply:
@Rineee ok I must have added another when correcting checkbox button position then messed up when trying to 'fix' it :p will undo and maybe squash all commit to clear it up a bit after I have the last configs from Marcelo